README.md

```diff
@@ -117,7 +117,7 @@ Defaults to `false`. Specifies if decimal character escapes should be used inste
 ```swift
 import HTMLEntities
 
-let text = "한, 한, ế, ế, 🇺🇸"
+let text = "한, 한, ế, ế, 🇺🇸"
```

Comment:
Can you double-check these unicode strings?  I'm assuming they weren't intended to be changed?

Reply:
Yeah they weren't intended to be changed. Don't think I intentionally touched them. They don't even look changed to me. Do you see anything?

Reply:
That's strange. It appears in every line where these kinds of scalars exist. Could it be a Safari bug?

Reply:
They're definitely different.  Github is highlighting the changed values.  Did you update this from GitHub.com or by checking out and editing?  Probably need to just undo those changes with your editor.  (Xcode can do line-level undos if you need).

Reply:
I did those from a browser, so GitHub.com

Reply:
Oh they are different! I'll fix this.

Reply:
Hmm... I think something is wrong over at my end. Possibly a localization issue on my machine. Can you push through the other changes and ignore this file?

Reply:
If you're able to pull the repo locally, you can use Xcode to undo specific lines of changes, then re-push your PR.